Kitodo/src/main/java/de/sub/goobi/forms/AktuelleSchritteForm.java

```diff
@@ -852,6 +853,21 @@ public void setMySchritt(Task task) {
         setStep(task);
     }
 
+    /**
+     * Get task with specific id.
+     *
+     * @param id passed as int
+     * @return task
+     */
+    public Task getTaskById(int id) {
+        try {
+            return serviceManager.getTaskService().getById(id);
+        } catch (DAOException e) {
+            logger.error(e.getMessage());
+            return null;
```

Comment:
1) Returning `null` should be avoided as it introduce unnecessary null checks and null pointer exceptions.
2) Should the user not be informed in this case? You are only logging it but did not inform the user.